libs/designsystem/modal/src/modal/modal-component/modal.component.ts

```diff
@@ -63,7 +63,7 @@ export class ModalComponent implements OnChanges, AfterContentInit {
   }
 
   // NOTE: input properties forwarded to modal-wrapper ModalConfig
-  @Input() collapseTitle = true;
+  @Input() collapseTitle = false;
```

Comment:
I assume this (inverting the default value) is intentional, but just in case: Is it intentional? I couldn't find a request for this change in the issue description (but I may have missed it 🙂)

Reply:
Good catch, it is intentional as requested by @henrikvoetmand during design review.
Setting it to false matches the controller-based modals default.